assets/js/modules/idea-hub/datastore/draft-ideas.js

```diff
@@ -38,6 +38,12 @@ const fetchCreateIdeaDraftPostStore = createFetchStore( {
 		return {
 			...state,
 			draftPostIdeas: [ ...( state.draftPostIdeas || [] ), ideaDraftPost ],
+			newIdeas: [
+				...( state.newIdeas || [] ),
+			].filter( ( { name } ) => name !== ideaDraftPost.name ),
+			savedIdeas: [
+				...( state.savedIdeas || [] ),
+			].filter( ( { name } ) => name !== ideaDraftPost.name ),
```

Comment:
No need to spread `newIdeas` and `saveIdeas`. You can call the `filter` function directly:
```suggestion
			newIdeas: ( state.newIdeas || [] ).filter( ( { name } ) => name !== ideaDraftPost.name ),
			savedIdeas: ( state.savedIdeas || [] ).filter( ( { name } ) => name !== ideaDraftPost.name ),
```

Reply:
Good point, thanks